### ISSUE TITLE
on up arrow, textarea is not synced, so screen reader user cannot review prior command

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#190999]
1. with a screen reader enabled, run some commands
2. hit enter
3. up arrow
4. use left and right arrow keys to inspect the previously run command
5. 🐛 you cannot
